include/picongpu/plugins/openPMD/openPMDDimension.hpp

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2014-2021 Axel Huebl, Felix Schmitt, Heiko Burau, Rene Widera,
+ *                     Franz Poeschel
```

Comment:
Should be a single line

Reply:
should be fine. we historically break into a new line if it gets too long.

Reply:
Yes this was copied out of the original file

Reply:
Note: In the update, I keep the author list identical to the source file (even the line break)